src/styles/index.scss

```diff
@@ -1,3 +1,209 @@
-body {
+@import './utils/variables';
+@import './blocks/page';
+@import './blocks/nav';
+@import './blocks/header';
+@import './blocks/stars';
+@import './blocks/buy';
+@import './blocks/card';
+
+// Variaveis
+
+$main-accent: rgba(6, 11, 53, 1);
+$secondary: rgba(97, 96, 112, 1);
+$white: rgba(255, 255, 255, 1);
+$blue-accent: rgba(0, 172, 220, 1);
+$gray-elements: rgba(243, 243, 243, 1);
+$transition-duration: 300ms;
+$gray-border: 1px solid $gray-elements;
+$title-hover: #34568b;
+$blue-border: 1px solid $blue-accent;
+$rounding: 5px;
+
+.buy {
+  text-align: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  padding: 12px 70px;
+  color: rgba(255, 255, 255, 1);
+  background-color: $blue-accent;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  border-radius: $rounding;
+  transition: $transition-duration;
+
+  &:hover {
+    background-color: $white;
+    color: $blue-accent;
+    border: $blue-border;
+  }
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(200px, 200px));
```

Comment:
The 'grid-template-columns' property is set to 'repeat(auto-fit, minmax(200px, 200px))'. Consider using a more flexible approach like 'minmax(200px, 1fr)' to allow the grid items to expand and fill the available space more effectively.